docs/source/markdown/podman-machine-init.1.md.in

```diff
@@ -24,9 +24,9 @@ but can be optionally used on Linux.
 SSH keys are automatically generated to access the VM, and system connections to the root account
 and a user account inside the VM are added.
 
-By default, the VM distribution is [Fedora CoreOS](https://getfedora.org/en/coreos?stream=testing) except for
-WSL which is based on a custom Fedora image.  While Fedora CoreOS upgrades come out every 14 days, the automatic
-update mechanism Zincata is disabled by Podman machine.
+By default, the VM distribution is a custom [Fedora CoreOS](https://getfedora.org/en/coreos?stream=stable)
+based image pushed to quay.io/podman/machine-os (build in https://github.com/containers/podman-machine-os).
+Except for WSL which is based on a custom Fedora image.
```

Comment:
"WSL is an exception and is based on a custom Fedora image"

Reply:
done